web/themes/joinup/templates/parts/rdf-entity--collection--listing-item--add-collection.html.twig

```diff
@@ -4,11 +4,5 @@
  * Themeing for rdf entities.
  */
 #}
-{%
-  set classes = [
-    'mdl-cell--middle'
-  ]
-%}
-<div class="{{ classes|join(' ') }}">
-  {{ add_collection_link }}
-</div>
+ 
+{{ add_collection_link }}
```

Comment:
Could it be possible maybe to remove somehow the template since it's only 1 print statement?